tools/build/binary-release/assets/Windows/README.txt

```diff
@@ -50,25 +82,83 @@ Microsoft BuildTools contain that compiler. You can use the installer script
 to guide you through the installation.
 
 Alternatively you can install the BuildTools manually. They can be downloaded
-[here](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools).
+[here](https://aka.ms/vs/stable/vs_BuildTools.exe).
 You'll need to select the `C++ build tools` and recommended components.
 
 The compiler is only usable in a special Build Tools CMD / PowerShell window.
+Once the BuildTools  are installed you should see new entries in the start
+menu:
+
+Start Menu -> Visual Studio 20XX -> x64 Native Tools Command Prompt for VS
+
+The `set-env` script automatically sets up the terminal for BuildTools as well.
+
+
+PATH
+====
+
+THERE IS MORE THAN ONE FOLDER YOU NEED TO ADD TO YOUR PATH!
+
+`raku` and it's associated programs are located in the `bin\` subfolder.
+Installed Raku scripts, including `zef`, are located in the
+`share\perl6\site\bin\` subfolder. So both need to be in your PATH.
+
+
+Preventing Garbled Text Output
+==============================
+
+First a bit of background. Windows supports two types of text encoding that
```

Comment:
A bit hard to read ... this block of text should be some combination of word-wrapped and broken into paragraphs with blank lines between them.